CMakeLists.txt

```
@@ -105,6 +105,9 @@ INSTALL_BUILD_STATS_SCRIPTS()
 # Install TriBITS so that other projects can use it
 include(SetupTribitsInstall)
 
+# See https://tribitspub.github.io/TriBITS/users_guide/index.html#project-name-must-find-all-tpl-libs
+SET(Trilinos_MUST_FIND_ALL_TPL_LIBS TRUE)
```

```suggestion
SET(Trilinos_MUST_FIND_ALL_TPL_LIBS_DEFAULT TRUE)
```

As per:

* https://tribitspub.github.io/TriBITS/users_guide/index.html#project-name-must-find-all-tpl-libs

There are several problems with setting an non-cache var like this when there is also a cache var of the same name.